struct ortentry and struct ifalias req are actually not defined
struct arpreq is defined, but can not be passed to an ioctl on the Hurd.
so let's make packages not believe these are available.

---
 sysdeps/mach/hurd/bits/ioctls.h |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/sysdeps/mach/hurd/bits/ioctls.h
+++ b/sysdeps/mach/hurd/bits/ioctls.h
@@ -304,12 +306,16 @@
 #define	SIOCGIFMETRIC	_IOWR('i',23, struct ifreq_int)	/* get IF metric */
 #define	SIOCSIFMETRIC	_IOW('i',24, struct ifreq_int)	/* set IF metric */
 #define	SIOCDIFADDR	_IOW('i',25, struct ifreq)	/* delete IF addr */
+#if 0
 #define	SIOCAIFADDR	_IOW('i',26, struct ifaliasreq)	/* add/chg IF alias */
+#endif
 
+#if 0
 #define	SIOCSARP	_IOW('i', 30, struct arpreq)	/* set arp entry */
 #define	OSIOCGARP	_IOWR('i',31, struct arpreq)	/* get arp entry */
 #define	SIOCGARP	_IOWR('i',38, struct arpreq)	/* get arp entry */
 #define	SIOCDARP	_IOW('i', 32, struct arpreq)	/* delete arp entry */
+#endif
 
 #define SIOCGIFMTU	_IOWR('i', 51, struct ifreq_int)/* get IF mtu */
 #define SIOCSIFMTU	_IOW('i', 52, struct ifreq_int)	/* set IF mtu */
